crates/lox_core/src/bodies/nutation/iau1980.rs

```diff
@@ -1,10 +1,27 @@
 use std::f64::consts::TAU;
 
-use crate::bodies::nutation::{point1_milliarcsec_to_rad, Coefficients, Nutation};
+use crate::bodies::nutation::{point1_milliarcsec_to_rad, Nutation};
 use crate::math::{arcsec_to_rad, normalize_two_pi};
 use crate::time::intervals::TDBJulianCenturiesSinceJ2000;
 use crate::types::{Arcsec, Radians};
 
+struct Coefficients {
+    /// Coefficients of l, l', F, D and Ω.
+    l: f64,
+    lp: f64,
```

Comment:
I see on line 27 the term `l_prime` is used. Perhaps use the same term in both places?